### DISCUSSION TITLE
fluentui/builder - evaluate usage of craft.js

### DISCUSSION THREAD

**Discussion by ankri** [microsoft/fluentui#16812] [Ideas] [answered]
Hi team,

during development of the fluentui builder, have you had the chance to evaluate [craft.js](https://github.com/prevwong/craft.js) a React Framework to build UI builders / page editors? It's a small project but could potentially be a good fit for the builder.

Full disclosure: I am a contributor and could help out with questions.

**Comment by JustSlone** [comment, marked as answer]
This looks interesting. Work on the UI Builder is currently on hold, but this might be interesting to consider later. I'm not sure if the editing experience is super core to the behavior of, but probably not as long as craft.js is extensible enough.
